scripts/launch-xrd

```diff
@@ -131,13 +131,27 @@ _runcmd() {
     fi
 }
 
+# Pulls an image if not loaded already. Return code 1 if image is unavilable
+#   Arg 1: image name
+_pull_image() {
+    if ! "$CONTAINER_EXE" pull "$1" || ! "$CONTAINER_EXE" inspect "$1" &>/dev/null; then
+        echo "Unrecognised image '$1' - should be either an image loaded " \
+            "in the local $CONTAINER_MGR registry or an image that can be pulled." >&2
+        return 1
+    fi
+}
+
 # Get the platform from the labels on a container image.
 #   Arg 1: image name
 _get_platform() {
     local platform
-    platform="$($CONTAINER_EXE inspect "$1" \
+    if ! platform="$($CONTAINER_EXE inspect "$1" \
         --format='{{index .Config.Labels "com.cisco.ios-xr.platform"}}' \
-        2>/dev/null)" || true
+        2>/dev/null)"; then
+        # This should never be hit.
+        echo "Image $1 has not be loaded in the local $CONTAINER_MGR registry." >&2
```

Comment:
I think the following probably makes a tiny bit more sense to me in general, and then later we'll follow up with a message about loading in an image
```suggestion
        echo "Image $1 cannot be found in the local $CONTAINER_MGR registry." >&2
```